importer/writer.py

```diff
@@ -59,17 +108,25 @@ def build_import_info(self):
         self.write_string(6, 'Total size of files in GBytes', self.spreadsheet.size)
         self.write_string(7, 'Data to be kept until', self.spreadsheet.limit)
 
-    def build_read_data(self, breakpoint: int):
+
+    def build_read_data(self, breakpoint: int, download):
         for read in range(breakpoint):
             position = read + 10
             end_reached = self.row == len(self.spreadsheet.reads)
             if end_reached:
                 self.status_closed = True
                 break
             current_row = self.spreadsheet.reads[self.row]
-            self.sheet.write(position, 0, current_row.forward_read)
-            if current_row.reverse_read is not None:
-                self.sheet.write(position, 1, current_row.reverse_read)
+            if download:
+                forward_read_file = current_row.forward_read + '_1.fastq.gz'
```

Comment:
I don't think this is correct. For single-ended data the file names on the ENA are just accession.fastq.gz